core/src/main/java/org/elasticsearch/indices/query/IndicesQueriesRegistry.java

```diff
@@ -39,4 +41,11 @@ public IndicesQueriesRegistry(Settings settings, Map<String, QueryParser<?>> que
     public Map<String, QueryParser<?>> queryParsers() {
         return queryParsers;
     }
+    
+    /**
+     * Returns all registered sort parsers
+     */
+    public Map<String, SortBuilderTemp<?>> sortParsers() {
+        return sortParsers;
+    }
```

Comment:
No - it's actually used here:

https://github.com/MaineC/elasticsearch/commit/313cdc045d046ab88374e6bfa91c4ddd0a167f34#diff-5e28e9c3ba33ef9b1a732c03df0a5f1aR152

I'm not sure I got your comment about registering all sort parsers under a common name correctly though. Maybe discuss later f2f when you have some bandwidth?